overlay/src/renderer/styles.css

```
@@ -142,3 +290,15 @@
 .state-error {
 	border: 1px solid rgba(239, 68, 68, 0.3);
 }
+
+.sr-only {
+	position: absolute;
+	width: 1px;
+	height: 1px;
+	padding: 0;
+	margin: -1px;
+	overflow: hidden;
+	clip: rect(0, 0, 0, 0);
```

Stylelint (17.6.0)

[error] Unexpected deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)